Makefile

```diff
@@ -116,7 +116,8 @@ config: certs ## Set up default config
 
 .PHONY: musl
 musl: ## Add target for musl
-	rustup target add $(uname_m)-unknown-linux-musl
+	rustup target list | grep -qc $(uname_m)-unknown-linux-musl || \
+	rustup target install $(uname_m)-unknow-linux-muslknown-linux-musl
```

Comment:
You can run `rustup target --help` to see the available subcommands. "install" is not an option.

Reply:
so it's rustup target add